apps/server/internal/runexec/docker_integration_test.go

```
@@ -0,0 +1,375 @@
+		if err := os.WriteFile(filepath.Join(repositoryPath, name), []byte("baseline\n"), 0o644); err != nil {
+			t.Fatal(err)
+		}
+	}
+	runIntegrationCommand(t, repositoryPath, "git", "add", ".")
+	runIntegrationCommand(t, repositoryPath, "git", "commit", "-qm", "baseline")
+	return repositoryPath
+}
+
+func runIntegrationCommand(t *testing.T, dir string, command ...string) {
+	t.Helper()
+	cmd := exec.Command(command[0], command[1:]...)
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)